website/common/locales/en/emptyArmory.json

```diff
@@ -0,0 +1,22 @@
+{
+    "emptyArmoryNew": "Empty the armory: New Challenge Available!",
+    "emptyArmoryUnlock": "You've unlocked 'Empty the Armory'! This special Market item allows you to reset your equipment. Use it to breathe new challenge into Habitica if you feel you've achieved it all, or if you just like running around naked!",
```

Comment:
We can't use "running around naked" because it's something of an adult topic. :)

Reply:
I know I know. Just sounded funny at the moment, thought would take it out later... but forgot, my bad :)

Reply:
It did sound funny. It made me smile. :)